### PR TITLE
Peter jubaer noel contact branch

### PR DESCRIPTION
**Pull request by noelfoka**
In this branch, I create contact section with:

- [ ] HTML
- [ ] CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,231 +1,298 @@
...
+                    placeholder="Write your message here" required></textarea>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
+            </form>
+        </div>
+    </section>
 
+<<<<<<< HEAD
+<<<<<<< HEAD
```
Comment (shella12) [inline index.html]: - [ ] [OPTIONAL] Great job on your `contact form` however your `contact form` and `about section` are repeating on your portfolio. This is probably caused by the conflicting code between branches that you might have tried to solve. I would suggest you  to remove all the code from line `294 - 371` and then removing line `374` as well


https://user-images.githubusercontent.com/44798044/229990444-ee33f3f2-7ad5-49e4-ae4f-cbbfef505d46.mov

File: index.html [hunk 2]
```diff
@@ -1,231 +1,298 @@
...
+                with it
+            </p>
+            <form action="https://formspree.io/f/xqkgeweb" method="POST" class="contact-form">
+                <input type="text" name="name" id="username" placeholder="Your Name" maxlength="30" required />
+                <input type="email" name="email" id="useremail" placeholder="Your Email" required />
+                <textarea name="message" id="usermessage" cols="30" rows="10" maxlength="500"
+                    placeholder="Write your message here" required></textarea>
+                <div class="button-resume btn-form">
+                    <button class="btn" type="submit">Get in touch</button>
+                </div>
```
Comment (shella12) [inline index.html]: - [ ] Please decrease the `margin-top ` and add a bit of `margin-left` into the contact form button to adjust the placement of the button to make it look like [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=7jqj8aDna8hNS8yv-0) as the aim is to make your design look as close to the figma design as possible. 

Figma design:

Screenshot 2023-04-05 at 10 40 26 AM

Your design:

Screenshot 2023-04-05 at 10 39 53 AM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @team, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- Professional README.md ✅   
- Correct use of git flow ✅ 
- Good commit history ✅ 
- Descriptive pull request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Used  Github flow :heavy_check_mark: 
Great job so far :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**